css/properties/text-size-adjust.json

```diff
@@ -62,16 +55,9 @@
             "opera_android": {
               "version_added": "41"
             },
-            "safari": [
-              {
-                "version_added": false
-              },
-              {
-                "prefix": "-webkit-",
-                "version_added": false,
-                "notes": "Instead of ignoring the <code>-webkit-text-size-adjust</code> property, a bug prevents desktop Safari users from zooming in or out. The bug was fixed after Safari 5."
-              }
-            ],
+            "safari": {
+              "version_added": false
```

Comment:
Have you verified that this isn't supported in Safari macOS? But it is supported in Safari iOS and Chrome macOS? It seems like unusual situation.

Reply:
I just checked, and that does seem to be the case:

<img width="311" alt="Screen Shot 2022-05-04 at 15 26 48" src="https://user-images.githubusercontent.com/5179191/166835518-b5c4975a-3c74-4fdc-9e52-5a4a387183f6.png">
(top is my laptop, bottom is my iPad)

Reply:
> Have you verified that this isn't supported in Safari macOS?

That's correct.